src/store/modules/auth/getters.ts

```diff
@@ -80,5 +83,5 @@ export const getters: GetterTree<State, IRootState> & Getters = {
   getTransactions: (state: State) => state.transactions,
   getExpense: (state: State)  => state.expense,
   getAuthFieldError: (state: State) => state.error,
-
+  getNotifications: (state: State) => state.notifications,
```

Comment:
return a the reverse of notifications array to show the latest notifications on the top.

Reply:
refactored